apps/microsoft-teams/app-actions/src/helpers/app-installation.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { parametersFromAppInstallation } from './app-installation';
+import { mockAppInstallation } from '../../test/mocks';
+import { AppInstallationParameters } from '../types';
+
+describe('parametersFromAppInstallation', () => {
+  it('returns the correctly typoed and formed paramaters from the app installation', () => {
```

Comment:
typo: change `typoed` tp `typed` and `paramaters` to `parameters`

Reply:
such an ironic typo 😬